pkg/create/templates/test.go

```diff
@@ -39,7 +39,7 @@ container_ip() {
 }
 
 run_sti_build() {
-  sti build ${sti_args} file://${test_dir}/test-app ${IMAGE_NAME} ${IMAGE_NAME}-testapp
+  sti build ${sti_args} ${test_dir}/test-app ${IMAGE_NAME} ${IMAGE_NAME}-testapp
```

Comment:
add file:/// back, it is the preferred way. The other one will give you fat warning.